src/Deprecated12/ClassOrganization.extension.st

```diff
@@ -44,13 +44,6 @@ ClassOrganization >> listAtCategoryNamed: aName [
 	^ self methodSelectorsInProtocol: aName
 ]
 
-{ #category : #'*Deprecated12' }
-ClassOrganization >> notifyOfAddedCategory: protocolName [
```

Comment:
I think this methods are really internal to the usage of ClassOrganization, and since we plan to kill it during Pharo 12, I don't think it's worth keeping the deprecation. 

But I can also keep the deprecated method without the transformation and explain why it was removed if people prefer that. 

But if someone is calling this method in their code, they are playing a really dangerous game now that I know better this part of the system...

Reply:
I do not think there is value to keep internal API method, especially when the whole MM is changed.